doc/source/reference/arrays.datetime.rst

```diff
@@ -98,8 +103,6 @@ because the moment of time is still being represented exactly.
     >>> np.datetime64('2005') == np.datetime64('2005-01-01')
     True
 
-    >>> np.datetime64('2010-03-14T15Z') == np.datetime64('2010-03-14T15:00:00.00Z')
-    True
```

Comment:
I probably missed the discussion elsewhere - why are we removing this?

Reply:
Because using timezone is not recommended, only supported on input for backwards compatibility, and raises a deprecation warning. See recommendation by @mattip at https://github.com/numpy/numpy/issues/14970#issuecomment-595485196

Reply:
I'd be inclined to keep the example but remove the `Z`s then

Reply:
makes sense

Reply:
Indeed, it passes without the Z.